src/models/transaction.ts

```diff
@@ -251,9 +251,7 @@ class Transaction {
     // Weight is a float with 8 bytes
     arr.push(helpers.floatToBytes(this.weight, 8));
     // Timestamp
-    if (this.timestamp) {
-      arr.push(helpers.intToBytes(this.timestamp, 4))
-    }
+    arr.push(helpers.intToBytes(this.timestamp!, 4))
```

Comment:
`this.timetamp!`? What's the exclamation mark at the end?

Reply:
The attribute `timestamp` is optional in the constructor (`null` by default). And `intToBytes` can't receive `null`, then I need to use the `!` to tell the linter that timestamp won't be null in this moment.

Reply:
Ok. Thanks!